Added basic docs (#280)
codatio/codat-docs: docs/assess/guides/migration-guides/deprecation-account-categories-portal.md

---
title: "Clients who are using Assess in the portal"
description: "Detailed instructions on how you can migrate from the features you're using today"
sidebar_label: "Clients using the portal"
createdAt: "2022-10-06T08:32:16.232Z"
updatedAt: "2022-11-10T14:01:08.149Z"
---

## What will happen on deprecation?
The following screens in **Assess** section of the portal will no longer be available:
- Profit and Loss (including Ratios)
- Balance sheet (including Ratios)
- Marketing
- Categorize accounts (Version 2 only)

> ![Screenshot of Profit and loss screen that's being deprecated in Assess](/img/assess/profit-and-loss-screen-assess.png)

Reviewer: all the screenshots are quite small and can't be zoomed into - is that ok that they're not very clear?

![Screenshot of Balance sheet screen that's being deprecated in Assess](/img/assess/balance-sheet-screen-assess.png)
![Screenshot of Marketing screen that's being deprecated in Assess](/img/assess/marketing-screen-assess.png)

The following screens in **Accounting API** section of the portal will no longer be available:
- Accounts - Categorize accounts

![Screenshot of categorize accounts screen that's being deprecated](/img/assess/categorize-accounts-button-accounting-api.png)
![Screenshot of categorize accounts screen that's being deprecated](/img/assess/categorize-accounts-screen1.png)
![Screenshot of categorize accounts screen that's being deprecated](/img/assess/categorize-accounts-screen2.png)

> ## Instructions for migration

Reviewer: is this really an instruction, or should the title be something to do with features?

| Feature 	| Alternative 	|  	|
|:--	|:--	|---	|

> | Under Assess: Balance sheet screen Profit and loss screen  Ratios (at the end of Profit and loss / Balance sheet screens) Marketing screen 	| We will no longer be supporting UIs for these however you can replicate these views with more flexibility in our new Excel report: Log into the Codat Portal → Select a company → Under products, click Assess → Reports A new ‘Assess Report' option is available to be generated and downloaded. Use pivot tables and calculations to generate the metrics and views that you need. 	|  	|

Reviewer: for this bit - Under Assess: Balance sheet screen Profit and loss screen Ratios (at the end of Profit and loss / Balance sheet screens
- I am not sure how this is meant to be organized, but it's really missing the line breaks

> | Under Assess: Balance sheet screen Profit and loss screen  Ratios (at the end of Profit and loss / Balance sheet screens) Marketing screen 	| We will no longer be supporting UIs for these however you can replicate these views with more flexibility in our new Excel report: Log into the Codat Portal → Select a company → Under products, click Assess → Reports A new ‘Assess Report' option is available to be generated and downloaded. Use pivot tables and calculations to generate the metrics and views that you need. 	|  	|

Reviewer: ```suggestion
| Under Assess: Balance sheet screen Profit and loss screen  Ratios (at the end of Profit and loss / Balance sheet screens) Marketing screen 	| We will no longer support UIs for these features. However, you can replicate them with more flexibility in our new Excel report. <br/> In the Codat Portal, navigate to **Companies** and select the relevant one. Under **Products**, click **Assess → Reports**, a new **Assess Report** option is available to be generated and downloaded. Use pivot tables and calculations to generate the metrics and views that you need. 	|  	|

```

> | Under Assess: Categorize accounts Under Accounting API Accounting Data → Accounts → Categorize accounts 	| If you want to recategorize accounts so the model learns and improves for your companies: Log into the Codat portal Select a company on the Companies page Click Categorize Accounts button Ensure the Version drop down on the right is set to Version 3 We have revamped this screen including some new features which make it easier to categorize accounts including Confidence level. If you are recategorizing accounts because the Enhanced Profit and Loss / Balance Sheet was not being returned due to uncategorized accounts: The latest version will return data regardless of whether all accounts have been categorized. You can decide using the confidence levels if you are happy proceeding without recategorizing. See example for how we suggest you interact with our new confidence levels. If you do not care about the model learning for your companies, you do not need to recategorize via Codat and can make amendments in your own environment. 	|  	|

Reviewer: ```suggestion
Use the confidence levels to decide if you want to proceed without recategorizing. See our example on how to interact with our new confidence levels. If you do not care about the model learning for your companies, you do not need to recategorize via Codat. Simply make amendments in your own environment. 	|  	|
```

also, where is this example you mentioned in the text?

|Feature|Portal|
|:--|:--|
|**Profit and loss**|![Screenshot of Profit and loss screen that's being deprecated in Assess](/img/assess/profit-and-loss.png)|
|**Balance sheet**|![Screenshot of Balance sheet screen that's being deprecated in Assess](/img/assess/balance-sheet.png)|
|**Ratios** <br/><br/>**Note**: These appear at the end of the Profit and Loss and Balance Sheet tabs|![Screenshot of Ratios screen that's being deprecated in Assess](/img/assess/financial-ratios.png)|
|**Marketing**|![Screenshot of Marketing screen that's being deprecated in Assess](/img/assess/marketing-screen-assess.png)|
|**Categorize accounts in Assess** <br/><br/>**Note**: We have new screens available for recategozing accounts using the latest categories.| ![Screenshot of categorize accounts screen that's being deprecated](/img/assess/account-categorization-v2.png)|
|**Categorize accounts in Accounting API** <br/><br/>**Note**: We have new screens available for recategozing accounts using the latest categories.| ![Screenshot of categorize accounts screen that's being deprecated](/img/assess/accounting-api-account-categorisation.png)|

## Alternative options

> ### Viewing financial statements and calculating ratios

Reviewer: full stops either everywhere, or nowhere

> 1. Log into the **Codat Portal**

Reviewer: ```suggestion
1. Log into the **Codat Portal**.

```

> 2. Select a company on the **Companies** page

Reviewer: ```suggestion
2. Select a company on the **Companies** page.

```

> 3. Under products, click **Assess**, and then **Reports**

Reviewer: ```suggestion
3. Under Products, click **Assess**, and then **Reports**.

```
